README.md

```diff
@@ -4,18 +4,628 @@
 
 _visit www.actionherojs.com for more information_
 
-This tutorial showcases a realistic Actionhero project. In addition to the `actionhero` project, it uses the [`ah-seqeulzie-plugin`](https://github.com/actionhero/ah-sequelize-plugin) to use models and migrations against a Postgres database.
+This tutorial showcases a realistic Todo List Application with CRUD operations.
 
-## To install:
+To simply play around with the application, follow the next steps:
 
-You will need node.js, npm, redis, and postgres available.
+1. Make sure you've the following installed on your system
+    - Redis
+    - Postgresql 
+    - NodeJS
 
-`npm install`
+2. Clone the project
 
-## To Run:
+```
+git clone https://github.com/actionhero/actionhero-tutorial.git 
+&& cd actionhero-tutorial
+```
 
-`npm start`
+3. Install the dependencies
+
+```
+npm install
+```
+
+4. Configure the database connection in `./src/config/sequelize.js` and run migrations.
+```
+npx sequelize-cli db:migrate
+```
+
+5. Start Dev server and access API's at <a href="http://localhost:8080">Localhost</a>
+```
+npm run test actions/task
+npm run dev
+```
+
+
+# Tutorial
+
+
+By the end of this tutorial you will become comfortable with the following pieces of Actionhero and build a CRUD based application.
+
+- Actions
+- Web Server Configs
+- Routing
+- Plugins
+- Testing
+
+We will be building a Todo List Web App which will cover all the basics to get started with Actionhero.
+
+<br>
+
+## Prerequisites
+
+- Make sure you've <a href="https://nodejs.org/en/" target="_blank">Nodejs</a> installed.
+
+- <a href="https://redis.io/" target="_blank">Redis</a> and <a href="https://www.postgresql.org/" target="_blank">Postgresql</a> are installed and working locally.
+
+<br>
+
+
+## Project Setup
+
+### Create a project directory:
+
+```
+mkdir todo && cd todo
+```
+
+Generate a new Actionhero project
+
+```
+npx actionhero generate
+```
+
+Install the Dependencies
+
+```
+npm install
+```
+
+Start the server by running:
+
+ `npm start` 
```

Comment:
It's probably better to get folks into the habit of running `npm run dev` instead - this way they don't need to worry about compiling their changes